vendor/bat-native-ads/README.md

```diff
@@ -80,7 +80,10 @@ void OnTimer(
 
 `OnUnidle` should be called periodically on desktop browsers as set by `SetIdleThreshold` to record when the browser is no longer idle. This call is optional for mobile devices
```

Comment:
Casing doesn't match the actual function name here.

Reply:
Fixed